integrations/serverless-integrating-service-mesh/serverless-integrating-ossm-3-x-setup.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-integrating-ossm-3-x-setup"]
+= Integrating Service Mesh 3.x with OpenShift Serverless
+include::_attributes/common-attributes.adoc[]
+:context: serverless-integrating-ossm-3-x-setup
+
+toc::[]
+
+[role="_abstract"]
+The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {SMProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. Integrating with Kourier disabled allows you to configure additional networking and routing options that the Kourier ingress does not support, such as mTLS functionality.
+
+Note the following assumptions and limitations:
+
+* All Knative internal components, as well as Knative Services, are part of the {SMProductShortName} and have sidecars injection enabled. This means that strict mTLS is enforced within the whole mesh. All requests to Knative Services require an mTLS connection, with the client having to send its certificate, except calls coming from OpenShift Routing.
+
+* {ServerlessProductName} with {SMProductShortName} integration can only target *one* service mesh. Multiple meshes can be present in the cluster, but {ServerlessProductName} is only available on one of them.
+
+[id="prerequisites_serverless-integrating-ossm-3-x-setup"]
+== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed the {ServerlessProductShortName} Operator.
+
+* You have installed the {SMProductName} 3.x Operator.
+
+* The examples in the following procedures use the domain `example.com`. The example certificate for this domain is used as a certificate authority (CA) that signs the subdomain certificate.
++
+To complete and verify these procedures in your deployment, you need either a certificate signed by a widely trusted public CA or a CA provided by your organization. Example commands must be adjusted according to your domain, subdomain, and CA.
+
+* You must configure the wildcard certificate to match the domain of your {ocp-product-title} cluster. For example, if your {ocp-product-title} console address is `https://console-openshift-console.apps.openshift.example.com`, you must configure the wildcard certificate so that the domain is `*.apps.openshift.example.com`. For more information about configuring wildcard certificates, see the following topic about _Creating a certificate to encrypt incoming external traffic_.
```

Comment:
```suggestion
* You must configure the wildcard certificate to match the domain of your {ocp-product-title} cluster. For example, if your {ocp-product-title} console address is `https://console-openshift-console.apps.openshift.example.com`, you must configure the wildcard certificate so that the domain is `*.apps.openshift.example.com`. For more information, see _Creating a certificate to encrypt incoming external traffic_.
```